pydocx/export/html.py

```diff
@@ -142,268 +152,711 @@ def style(self):
                 convert_dictionary_to_style_fragment(definition),
             ))
 
-        return self.make_element(
-            tag='style',
-            contents=''.join(result),
-        )
+        tag = HtmlTag('style')
+        return tag.apply(''.join(result))
+
+    def meta(self):
+        yield HtmlTag('meta', charset='utf-8', allow_self_closing=True)
+
+    def export(self):
+        return ''.join(
+            result.to_html() if isinstance(result, HtmlTag)
+            else result
+            for result in super(PyDocXHTMLExporter, self).export()
+        )
+
+    def export_document(self, document):
+        tag = HtmlTag('html')
+        results = super(PyDocXHTMLExporter, self).export_document(document)
+        return tag.apply(chain(self.head(), results))
+
+    def calculate_numbering_spans(self, paragraphs):
```

Comment:
Can you add a comment or a docstring at the top of this guy that informs the reader what each of the key names mean (active, open-level, etc)? I think I have a pretty good idea, but I want to be sure. And if I could use some help here, someone who doesn't know you or pydocx will most definitely need it :)